### PR TITLE
add post and shortlink for save resume clarification

### PR DESCRIPTION
**Pull request by jackgerrits**

### CODE REVIEW

File: _posts/2022-03-07-save-resume-clarification.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: "Clarification on save_resume defaulting to true in 9.0.0"
+layout: blog
+tags:
+description: More information on the same_resume change
+author: Jack Gerrits
+avatar_link: https://avatars1.githubusercontent.com/u/7558482?s=400&u=21e4cca683799d65a20a4cf3d11d0c17853ef9cb&v=4
+---
+
+For version 9.0.0 the [decision was made](https://github.com/VowpalWabbit/vowpal_wabbit/issues/3163) to change the default behavior of model saving to include information required to continue training from a loaded model. This was done as it seemed that most users assumed that this was already the default behavior leading to unexpected behavior. However, in particular when parsing `--invert_hash` output this change of default can be surprising to users. When `save_resume` is enabled `--invert_hash` includes the resume information in the readable format.
```
Comment (rajan-chari) [inline _posts/2022-03-07-save-resume-clarification.md:10]: With the introduction of Version 9.0.0, saved vw models allow online training to continue by default.  This is different from previous versions where --save_resume had to be explicitly specified.

Comment (rajan-chari) [reply _posts/2022-03-07-save-resume-clarification.md:10]: This change was motivated by a frequent failure mode in a common use case.  Specifically, when experimenting with VW models in Jupyter notebooks, train, save, and test is called interactively and repeatedly.  Failure to specify "save_resume" in the older version leads to unexpected behavior.

File: _posts/2022-03-07-save-resume-clarification.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---
+title: "Clarification on save_resume defaulting to true in 9.0.0"
```
Comment (rajan-chari) [inline _posts/2022-03-07-save-resume-clarification.md:2]: Title:  Advisory on using save_resume, invert_hash and readable_model with VW 9.0.0

File: _posts/2022-03-07-save-resume-clarification.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+layout: blog
+tags:
+description: More information on the same_resume change
+author: Jack Gerrits
+avatar_link: https://avatars1.githubusercontent.com/u/7558482?s=400&u=21e4cca683799d65a20a4cf3d11d0c17853ef9cb&v=4
+---
+
+For version 9.0.0 the [decision was made](https://github.com/VowpalWabbit/vowpal_wabbit/issues/3163) to change the default behavior of model saving to include information required to continue training from a loaded model. This was done as it seemed that most users assumed that this was already the default behavior leading to unexpected behavior. However, in particular when parsing `--invert_hash` output this change of default can be surprising to users. When `save_resume` is enabled `--invert_hash` includes the resume information in the readable format.
+
+Therefore as of version 9.0.0 `vw` now behaves as if `--save_resume` was passed to each invocation. To use the previous default behavior pass `--predict_only_model`.
```
Comment (rajan-chari) [inline _posts/2022-03-07-save-resume-clarification.md:12]: -- Para
As a result, additional information is now serialized when '--invert_hash' or ''readable_model' is specified. In order to get the old behavior, please use '--predict_only_model' when using '--invert_hash' or '--readable_model'
-- Para

Comment (rajan-chari) [reply _posts/2022-03-07-save-resume-clarification.md:12]: TLDR:  With VW 9.0.0, use '--predict_only_model' when using '--invert_hash' or '--readable_model' to get old behavior.

File: _posts/2022-03-07-save-resume-clarification.md (hunk 2 above)
Comment (jackgerrits) [reply _posts/2022-03-07-save-resume-clarification.md:2]: I feel "advisory" has negative connotations usually centered around "security advisories". This is certainly not something like that but  just clarifying what may have been missed or is confusing.

What about: `Clarification on using save_resume, invert_hash and readable_model with VW 9.0.0`

**Pull request merged**